### ISSUE TITLE
training set image and landmark annotation

### ISSUE THREAD

**Issue by wjtan99** [1adrianb/face-alignment-training#7]
First of all, thanks a lot for your code. 

I followed your instruction to prepare the data set for training.  I downed the landmark file, there are 4 subdirectories called AFW, HELEN, IBUG, and LFPW.   

I also followed the instruction on your homepage and downloaded the dataset LS3D-W, while your post says I should download  300W-LP. In the LS3D-W, there is no directory called  300W-LP, neither directories called the names in the landmarks directory.   Where can I downland the data set  300W-LP?  

Thanks a lot for your help.

**Comment by 1adrianb** [comment]
Hi @wjtan99 you can download the 300W-LP dataset from here: http://www.cbsr.ia.ac.cn/users/xiangyuzhu/projects/3DDFA/main.htm

**Comment by wjtan99** [comment]
I also saw in you LS

**Comment by wjtan99** [comment]
I also saw in all image directories of LS3D-W, there are .t7 file for every image.  Are those predicted landmarks or ground truth?  Can I use them as training set?   Thanks.

**Comment by 1adrianb** [comment]
@wjtan99 yes you can use them for training. They were obtained using the 2D-to-3D stack starting from the ground truth bounding boxes. I think there a few (like very few) with some wrong labels but this should not affect your results since I used them in the current form for the evaluations done in the paper.
I will sanitise them anyway toward the end of this month.
